framework/decode/file_processor.h

```diff
@@ -1,6 +1,7 @@
 /*
 ** Copyright (c) 2018 Valve Corporation
 ** Copyright (c) 2018 LunarG, Inc.
+** Copyright (c) 2023 Advanced Micro Devices, Inc. All rights reserved.
 **
```

Comment:
Interesting question: since you deleted a line below, and didn't add a line, should you add a copyright notice here??? 🤔

Reply:
The change has been reverted.